packages/react/src/components/InAppMessaging/BannerMessage/BannerMessage.tsx

```diff
@@ -0,0 +1,84 @@
+import isEmpty from 'lodash/isEmpty';
+
+import { Button } from '../../../primitives/Button';
+import { Flex } from '../../../primitives/Flex';
+import { Heading } from '../../../primitives/Heading';
+import { IconClose } from '../../../primitives/Icon/icons/IconClose';
+import { Image } from '../../../primitives/Image';
+import { Text } from '../../../primitives/Text';
+import { Link } from '../../../primitives/Link';
+import { useThemeBreakpoint } from '../../../hooks/useThemeBreakpoint';
+
+export default function BannerMessage({
+  alignment = 'right',
+  ...props
+}): JSX.Element | null {
+  const {
+    body,
+    header,
+    image,
+    onClose,
+    position,
+    primaryButton,
+    secondaryButton,
+  } = props;
```

Comment:
Think we can just destructure these all below `alignment` and remove the spread `props` 😉